### PR TITLE
Added some linux commands

### PR DESCRIPTION
**Pull request by i-m-arin**
Added some linux commands

### CODE REVIEW

**Comment by ghost** [comment]
Pls resolve conflict

**Comment by i-m-arin** [comment]
okay

**Comment by i-m-arin** [comment]
I have resolved the conflict.Please merge it.Thank you.

File: Linux.md [hunk 1]
```diff
@@ -67,6 +67,19 @@
 ### <a name="findAFile"></a> Find a file in Linux
 `$ sudo find . -name <file_name>`
 
+
+### Check IP to whitelist
+`curl ifconfig.me ; echo`
+
+### Change the ownership of a file
+`sudo chown new_owner_name filename`
```
Comment (ghost) [inline Linux.md]: Can you pls give one use-case in which scenario one will be compelled to use this command?

File: Linux.md [hunk 2]
```diff
@@ -67,6 +67,19 @@
...
+
+### Change the group to which a file belong
+`sudo chgrp new_group_name filename`
+
+### Change password for a user named mike
```
Comment (ghost) [inline Linux.md]: pls remove this command from the list

**Review by ghost — CHANGES_REQUESTED** [review]

File: Linux.md [hunk 3]
```diff
@@ -67,6 +67,10 @@
 ### <a name="findAFile"></a> Find a file in Linux
 `$ sudo find . -name <file_name>`
 
+
+### Check IP to whitelist
```
Comment (ghost) [inline Linux.md]: This command is already present in the doc at the line 74.

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**